public/style.css

```
@@ -0,0 +1,26 @@
+@font-face {
+    font-family: "cubanoregular";
+    src: url("https://thecodingtrain.com/assets/fonts/cubano-regular-webfont.woff2");
+}
+body {
+    display: flex;
+    flex-direction: column;
+    place-content: center;
+    place-items: center;
+    background-color: rgb(255,255,255);
```

```suggestion
    background-color: rgb(0,255,0);
```
This is for the chroma-key overlay